src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg

```diff
@@ -0,0 +1 @@
 id="rect15" class="st3" d="M132.9 33h34v31.5h-34z"/><path id="rect17" class="st4" d="M132.9 33h12v31.5h-12z"/><ellipse id="ellipse19" class="st4" cx="144.6" cy="40.8" rx="8" ry="10"/><path id="path21" class="st5" d="M132.1 29.4c0-9.7 7.9-17.6 17.6-17.6 9.7 0 17.6 7.9 17.6 17.6"/><path id="path23" class="st6" d="M149.8 11.8c9.6 0 17.4 7.8 17.4 17.5v.1h-17.4V11.8z"/><path id="rect25" class="st5" d="M130.9 28.8h37.4c1.3 0 2.3 1 2.3 2.3 0 1.3-1 2.3-2.3 2.3h-37.4c-1.3 0-2.3-1-2.3-2.3 0-1.3 1.1-2.3 2.3-2.3z"/><path id="polygon27" class="st7" d="M128 161.5l4.9-97.3h34l4.4 92.5z"/><path id="line29" class="st8" d="M167.4 71.7l1.3 25.8"/><path id="polygon31" class="st9" d="M131.6 88.4l-1.3 27 38.4-17.9-1.3-25.8z"/><path id="line33" class="st8" d="M170 122.8l1.3 25.8"/><path id="polygon35" class="st9" d="M128.9 142l-1.4 27 43.8-20.4-1.3-25.8z"/><path id="rect39" class="st6" d="M127.7 49.9h43.9v14.4h-43.9z"/><path id="rect41" class="st5" d="M127.7 49.9h28.8v14.4h-28.8z"/><path id="path43" class="st10" d="M149.3 49.9c0 5.2-4.9 9.4-10.8 9.4s-10.8-4.2-10.8-9.4"/><g id="layer1"><path id="path71" class="st2" d="M205 45.9c1.2 0 2.3.3 3.3.7a13.9 13.9 0 0 1 26.8-3.6 9.3 9.3 0 1 1 0 18.6H205c-4.4 0-7.9-3.5-7.9-7.9s3.5-7.8 7.9-7.8z"/><path id="path77" class="st11" d="M115.5 121.1c1 0 1.9.2 2.8.6.6-6.5 6.3-11.2 12.7-10.7 4.5.4 8.4 3.4 10 7.6 4.4-.2 8.1 3.1 8.3 7.5.2 4.4-3.1 8.1-7.5 8.3h-26.2c-3.7-.1-6.6-3.3-6.4-6.9 0-3.5 2.8-6.3 6.3-6.4z"/></g><g id="layer2"><circle id="circle45" class="st12" cx="180.9" cy="30.5" r="1.4"/><circle id="circle47" class="st13" cx="193.4" cy="110.1" r="2.5"/><circle id="circle49" class="st13" cx="44.4" cy="87.6" r="2.5"/><circle id="circle51" class="st13" cx="84.5" cy="130.4" r="2.5"/><circle id="circle53" class="st13" cx="182.8" cy="70.8" r="2.5"/><circle id="circle55" class="st13" cx="234.6" cy="134.1" r="2.5"/><circle id="circle57" class="st12" cx="39.1" cy="143.3" r="1.4"/><circle id="circle59" class="st12" cx="52.2" cy="114.7" r="1.4"/><circle id="circle61" class="st12" cx="61" cy="107.3" r="1.4"/><circle id="circle63" class="st12" cx="98.1" cy="74.2" r="1.4"/><circle id="circle65" class="st12" cx="215.4" cy="78.2" r="1.4"/><circle id="circle67" class="st12" cx="271.1" cy="137.4" r="1.4"/><circle id="circle69" class="st12" cx="74.6" cy="69.9" r="1.4"/><circle id="circle73" class="st12" cx="199.8" cy="138" r="1.4"/><circle id="circle75" class="st12" cx="242.7" cy="102.5" r="1.4"/></g><g id="layer3"><path id="path11" class="st10" d="M133.5 32.5S67 23.2 66.8 23.2c-26 17-38 35-44 44L133.4 51l.1-18.5z"/></g><g id="layer6"><path id="polygon37" class="st14" d="M168.2 64.2H151v88.1l20.7 9.2z"/></g><g id="layer5"><path id="path79" class="st15" d="M108.7 176.6c21 0 21-13.6 42-13.6s21 13.6 42 13.6c17.8 0 20.5-9.8 33.8-12.8-18.1-9.9-45.1-16.2-75.3-16.2S93.6 154 75.5 164c12.8 3.1 15.6 12.6 33.2 12.6z"/><path id="path81" class="st16" d="M108.8 176.4c21 0 21-13.6 42-13.6h.4l-.2-15c-30 0-57 6.2-75 16 12.5 3.2 15.4 12.6 32.8 12.6z"/></g></g></g></svg>
```

Comment:
please use the existing lighthouse logo, we don't need to upload another one.

Reply:
In this case, this is a version of the Lighthouse logo that includes additional treatments from our illustrator to match the outline styles found in the infographic. 

Compare new one: https://github.com/malchata/WebFundamentals/blob/4fb3722f559f7590a67596631a60089dd2d1825e/src/content/en/fundamentals/performance/speed-tools/images/tool-lighthouse.svg

to existing logo:
https://developers.google.com/web/progressive-web-apps/images/pwa-lighthouse.png